doc/source/ray-core/patterns/global-variables.rst

```diff
@@ -0,0 +1,30 @@
+Anti-pattern: Accessing or modifying global variables in tasks and actors
```

Comment:
@stephanie-wang: do you have suggestions to the title to make it more like others: doing something causes/harms xxx

Reply:
Hmm I guess we don't have a good "consequence" since it's really just unintuitive effect compared to single-process Python.

But how about updating to be more specific?
"Using global variables to share state between tasks and actors"